cron/errorAbandonedJobs.js

```diff
@@ -1,12 +1,7 @@
-var ERR = require('async-stacktrace');
+const util = require('util');
 
-var serverJobs = require('../lib/server-jobs');
-
-module.exports = {};
+const serverJobs = require('../lib/server-jobs');
 
 module.exports.run = function (callback) {
```

Comment:
Should we introduce `runAsync` support for cron jobs?

Reply:
I'm definitely on board if you want to add it.

Reply:
I do, but I'll save this for a separate PR.